tools/sigclient/Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY go.mod go.sum ./
 RUN go mod download
 
 COPY . .
-WORKDIR /usr/app/tools/sigclient
+WORKDIR /usr/app
 RUN go build -o sigclient main.go
 
-CMD ["./sigclient"]
\ No newline at end of file
+ENTRYPOINT ["./sigclient"]
```

Comment:
I'm not too familiar with CMD vs ENTRYPOINT. Is this a best-practices change, or is something deeper going on here?

Reply:
Yes, so if I we keep this as `CMD`, then if we keep some command like `ingest esbulk ...` then those will override the `CMD` instead of appending to it. 

But `ENTRYPOINT` will always have `./sigclient` as the start command and whatever other arguments will be appended to it.

And since we have a lot of different variation of args, I thought this would be better.

Reply:
Cool, sounds good